doc/spire_server.md

```diff
@@ -159,11 +159,13 @@ server {
         bundle_endpoint {
             address = "0.0.0.0"
             port = 8443
-            acme {
-                domain_name = "example.org"
-                email = "mail@example.org"
-            }
             refresh_hint = "10m"
+            profile "https_web" {
```

Comment:
non-blocking: Sorry I missed this before, but I think we need to document this in `### Configuration options for federation.bundle_endpoint` section?